tests/nlu/featurizers/test_convert_featurizer.py

```diff
@@ -58,18 +62,30 @@ def test_convert_featurizer_train(component_builder):
     )
 
     seq_vecs, sent_vecs = message.get_dense_features(TEXT, [])
+    if seq_vecs:
```

Comment:
why do we have to check for this? It's a test, we should be knowing what we get, right?

Reply:
same in the other files where these lines where added